capitoli/Nono_Passo_Liste.rst

```diff
@@ -153,7 +154,7 @@ Indici
 Si accede agli elementi di una lista nel modo già visto per accedere ai caratteri di una stringa. 
 ::
 
-	print (dtabellina [1]) Visualizza il secondo elemento della lista: tabellina
+	print (dtabellina [1]) #Visualizza il secondo elemento della lista: tabellina
```

Comment:
perchè dtabellina?

Reply:
Nel libro è riportato "dtabellina", lo modifico in "tabellina"?

Reply:
Si, sarà un refuso di stampa, metti `tabellina`. Grazie